### PR TITLE
Changed 1 character. Should fix stuff

### LINKED ISSUE #11: AssertionError: Number must be an unsigned integer, not a float
as discussed in https://github.com/adafruit/Adafruit_CircuitPython_RSA/pull/10, the simpletest for this library fails:

on 5.0
```
Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.
code.py output:
Generating keypair...
Encrypting message...
Traceback (most recent call last):
  File "code.py", line 16, in <module>
  File "/lib/adafruit_rsa/pkcs1.py", line 173, in encrypt
  File "/lib/adafruit_rsa/transform.py", line 181, in int2bytes
AssertionError: Number must be an unsigned integer, not a float.
```

and on 4.1.2
```
Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.
code.py output:
Generating keypair...
Encrypting message...
Traceback (most recent call last):
  File "code.py", line 16, in <module>
  File "/lib/adafruit_rsa/pkcs1.py", line 173, in encrypt
  File "/lib/adafruit_rsa/transform.py", line 177, in int2bytes
AssertionError: Number must be an unsigned integer, not a float.```
```

### PR DESCRIPTION
**Pull request by evaherrada**
Simpletest currently works. I'm running rsa_tests right now, and it's running better than it was before, although I'm not convinced this was the only issue.

### CODE REVIEW

File: adafruit_rsa/transform.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def int2bytes(number, fill_size=None, chunk_size=None, overflow=False):
         raise ValueError("You can either fill or pad chunks, but not both")
 
     # Ensure these are integers.
-    assert number & 1 == 0, "Number must be an unsigned integer, not a float."
+    assert number % 1 == 0, "Number must be an unsigned integer, not a float."
```
Comment (brentru) [inline adafruit_rsa/transform.py]: Might want to do `isinstance(number, int)` to verify the type, instead of performing modulo div

**Review by brentru — CHANGES_REQUESTED** [review]

**Comment by evaherrada** [comment]
It seems like there are issues with signing using sha based on the test file. I don't think the issues are related to this one. I haven't gotten to the end of the test file yet but I'll just use a try/except to find out how each one is failing. I'll open an issue, and if it seems easy enough, fix it.

**Comment by brentru** [comment]
@dherrada  Could you attach the REPL output of `simpletest` with this patch applied?

**Comment by evaherrada** [comment]
@brentru sure. Here ya go
```
code.py output:
Generating keypair...
Encrypting message...
Decrypting message...
Decrypted Message:  hello blinka
```

**Review by brentru — APPROVED** [review]
Looks good and fixes assertion issues

**Pull request merged**